flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/NXLocalApi.js

```diff
@@ -0,0 +1,17 @@
+class NXLocalApi {
```

Comment:
Where is this company route stuff handled in the real A/C?
I would assume that it is part of the ATSU, isn't it?

Reply:
Navigation database. They are loaded through the data loading selector.